frontend-ssr/components/tableOfContents/TableOfContents.module.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../styles/variables';
+@import '../../styles/buttons';
+.tableOfContents {
+    li {
```

Comment:
`ul` elements come with a default padding. Let's remove it. I think the best way is to assume we always remove this for all `ul/ol` elements, so you can add this style in `index.scss` for now.